src/utils/chip.ts

```diff
@@ -0,0 +1,51 @@
+export function getChipColor(
```

Comment:
i think you better move this function into useChipController (create a new hooks)